doc/postman/device-onvif-camera.postman_collection.json

```diff
@@ -3794,7 +3794,7 @@
 						"method": "GET",
 						"header": [],
 						"url": {
-							"raw": "http://0.0.0.0:59882/api/v3/device/name/{{EDGEX_DEVICE_NAME}}/Snapshot",
+							"raw": "http://0.0.0.0:59882/api/v3/device/name/{{EDGEX_DEVICE_NAME}}/Snapshot?jsonObject={{EDGEX_MEDIA_PROFILE_BASE64}}",
```

Comment:
@presatish , @ajcasagrande , this is why the SnapShot resource is working. I.e. the profile is being passed manually.

That Device Command can be removed.

Reply:
Yes that was the point of this PR :)